frontend/public/components/nav/nav-header.tsx

```diff
@@ -55,25 +66,46 @@ const NavHeader_: React.FC<NavHeaderProps & StateProps> = ({ onPerspectiveSelect
     [isPerspectiveDropdownOpen, togglePerspectiveOpen],
   );
 
-  const perspectiveItems = React.useMemo(
-    () =>
-      perspectiveExtensions.map((nextPerspective: Perspective) => (
+  const perspectiveItems = React.useMemo(() => {
+    const items = perspectiveExtensions.map((nextPerspective: Perspective) => (
+      <DropdownItem
+        key={nextPerspective.properties.id}
+        onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
+          onPerspectiveSelect(event, nextPerspective)
+        }
+        isHovered={nextPerspective.properties.id === activePerspective}
+      >
+        <Title headingLevel="h2" size="md" data-test-id="perspective-switcher-menu-option">
+          <span className="oc-nav-header__icon">{nextPerspective.properties.icon}</span>
+          {nextPerspective.properties.name}
+        </Title>
+      </DropdownItem>
+    ));
+    if (acmLink) {
+      items.push(
         <DropdownItem
-          key={nextPerspective.properties.id}
-          onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
-            onPerspectiveSelect(event, nextPerspective)
-          }
-          isHovered={nextPerspective.properties.id === activePerspective}
-          component="button"
+          key={ACM_LINK_ID}
```

Comment:
Why is this not a `Perspective` extension like we do for admin and dev perspectives?

Reply:
I believe that was discussed and this direction taken.  I don't recall the details.  @spadgett ?

Reply:
fixed, thanks

Reply:
I'd like to know the reason. I think going against the extension mechanism is a bad idea.

Reply:
I'll +1 @christianvogt on this. As stated in my review, I also feel like we should be using the Perspective extension mechanism here.

Reply:
@christianvogt Can you add an external link using the Perspective extension? It's not a normal perspective, but a link out to another console.

The other problem is that the URL is not static, but has to be discovered from a `ConsoleLink` resource which we're watching.

Reply:
I see... I concede then.
Is this a short term link out? It's odd that we're mixing concepts in this dropdown.

Reply:
Hi @spadgett, Can we move to Perspective extension for ACM phase 2 where both OCP and ACM behind same url?  Might still have to get URL from ACM's `ConsoleLink` but it wont be an external link, right?

Reply:
The current phase 2 designs remove the perspective and instead add a cluster picker with an "All Clusters" option that goes to ACM. So this is a short-term link.